docs/content/preview/yugabyte-platform/troubleshoot/install-upgrade-issues/kubernetes.md

```diff
@@ -0,0 +1,309 @@
+
+- Follow the well-written [Troubleshooting guide for EBS volumes](https://aws.amazon.com/premiumsupport/knowledge-center/eks-troubleshoot-ebs-volume-mounts/) from AWS.
+
+## Fix ImagePullBackOff/ErrImagePull Error
+
+**Symptom**
+
+Kubernetes pods sometimes encounter this issue when it fails to pull the container images from a private container registry. If an error occurs during the pull, the pod goes into the `ImagePullBackOff` state. For more information, see [ImagePullBackOff](https://kubernetes.io/docs/concepts/containers/images/#imagepullbackoff).
+
+The `ImagePullBackOff` state can occur due to the following reasons -
+1. The image path is incorrect.
+2. The network fails - unable to pull image due to network limitation.
+3. The `kubelet` does not succeed in authenticating with the container registry.
+
+```sh
+## Initially, you will see ErrImagePull, but after trying multiple times, you will see ImagePullBackOff.
+## NAME                 READY   STATUS                  RESTARTS   AGE
+## yw-test-yugaware-0   0/4     Init:ErrImagePull       0          3
+
+## NAME                 READY   STATUS                  RESTARTS   AGE
+## yw-test-yugaware-0   0/4     Init:ImagePullBackOff   0          2m10s
+```
+
+**Debug**
+
+- Describe the pod to debug it further.
+
+  ```sh
+  kubectl describe pod yw-test-yugaware-0 -n <NAMESPACE>
+
+  ## output
+  ## Events:
+  ##   Type     Reason                  Age                 From                     Message
+  ##   ----     ------                  ----                ----                     -------
+  ##   Normal   Pulling                 25s (x3 over 75s)   kubelet                  Pulling image "quay.io/yugabyte/yugaware:2.16.0.0-b90"
+  ##   Warning  Failed                  22s (x3 over 72s)   kubelet                  Failed to pull image "quay.io/yugabyte/yugaware:2.16.0.0-b90": rpc error: code = Unknown desc = failed to pull and unpack image "quay.io/yugabyte/yugaware:2.16.0.0-b90": failed to resolve reference "quay.io/yugabyte/yugaware:2.16.0.0-b90": pulling from host quay.io failed with status code [manifests 2.16.0.0-b90]: 401 UNAUTHORIZED
+  ##   Warning  Failed                  22s (x3 over 72s)   kubelet                  Error: ErrImagePull
+  ##   Normal   BackOff                 7s (x3 over 72s)    kubelet                  Back-off pulling image "quay.io/yugabyte/yugaware:2.16.0.0-b90"
+  ##   Warning  Failed                  7s (x3 over 72s)    kubelet                  Error: ImagePullBackOff
+  ```
+
+**Resolution**
+
+- In case of **Bad pull secret / no pull secret / Bad pull secret name** - You need a pull secret to fetch the images from Yugabyte Quay.io and ensure you have applied the same in the namespace, which will use to install YBA. It will default look for the following named `yugabyte-k8s-pull-secret` secret. For more information, see [values.yaml](https://github.com/yugabyte/charts/blob/24a8dcf3a4c33153477e3e3ba82f9f4b6e2967a5/stable/yugaware/values.yaml#L16).
```

Comment:
```suggestion
- In case of **Bad pull secret / no pull secret / Bad pull secret name** - You need a pull secret to fetch the images from Yugabyte Quay.io registry and ensure you have applied the same in the namespace, which will be used to install YBA. It will look for a secret with name `yugabyte-k8s-pull-secret` by default. For more information, see [values.yaml](https://github.com/yugabyte/charts/blob/24a8dcf3a4c33153477e3e3ba82f9f4b6e2967a5/stable/yugaware/values.yaml#L16).
```